### DISCUSSION TITLE
关于jsDoc注释（api篇） | LeeDaisen

### DISCUSSION THREAD

**Discussion by giscus** [admin8756/leelaa-blog-comments#9] [Announcements]
# 关于jsDoc注释（api篇） | LeeDaisen

如何优雅的写出js的注释-api篇

https://leelaa.cn/posts/JavaScript/%E5%85%B3%E4%BA%8EjsDoc%E6%B3%A8%E9%87%8A.html

**Comment by ATQQ** [comment]
1
